source/applications/basic/file_formats/read_zdf_without_zivid.py

```diff
@@ -1,16 +1,18 @@
 """
 Import ZDF point cloud without Zivid Software.
+Note: ZIVID_DATA needs to be set to the location of Zivid Sample Data files.
 """
 
 from pathlib import Path
+import os
```

Comment:
not needed if we do `zivid.environment.data_path()` instead of `os.environ['ZIVID_DATA'])` right?